tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Grpc/GrpcLegacy/Client/GrpcLegacyClientCommon.cs

```diff
@@ -242,20 +242,13 @@ private static Span CreateInactiveSpan(Tracer tracer, string? methodFullName)
             var operationName = tracer.CurrentTraceSettings.Schema.Client.GetOperationNameForProtocol("grpc");
             var serviceName = tracer.CurrentTraceSettings.Schema.Client.GetServiceName(component: "grpc-client");
             var tags = tracer.CurrentTraceSettings.Schema.Client.CreateGrpcClientTags();
-            var span = tracer.StartSpan(operationName, tags, serviceName: serviceName, addToTraceContext: false);
             tags.SetAnalyticsSampleRate(IntegrationId.Grpc, tracer.Settings, enabledWithGlobalSetting: false);
             tracer.CurrentTraceSettings.Schema.RemapPeerService(tags);
 
+            var span = tracer.StartSpan(operationName, tags, serviceName: serviceName, addToTraceContext: false);
```

Comment:
Was this moved for a technical reason, or just to keep all the "tag setting" stuff together? (If the latter, all good, if the former, maybe we should add a comment about why)

Reply:
Just to keep the "tag setting" and "span setting" lines together 🧹  (sorry, I couldn't help myself).

Reply:
I often make a lot of "clean up" changes which I regret later when looking at the PR and rever them. I must have missed this one 😅